config.go

```diff
@@ -28,12 +28,14 @@ const (
 	SettingListen        = "listen"
 	SettingListenDefault = ":8080"
 
-	SettingsAws               = "aws"
-	SettingAwsS3Region        = SettingsAws + ".region"
-	SettingAwsS3RegionDefault = "us-east-1"
-	SettingAwsS3Bucket        = SettingsAws + ".bucket"
-	SettingAwsS3BucketDefault = "mender-artifact-storage"
-	SettingAwsURI             = SettingsAws + ".uri"
+	SettingsAws                   = "aws"
+	SettingAwsS3Region            = SettingsAws + ".region"
+	SettingAwsS3RegionDefault     = "us-east-1"
+	SettingAwsS3Bucket            = SettingsAws + ".bucket"
+	SettingAwsS3BucketDefault     = "mender-artifact-storage"
+	SettingAwsURI                 = SettingsAws + ".uri"
+	SettingsAwsTagArtifact        = SettingsAws + ".tag_artifact"
```

Comment:
@mchalski fyi needed to go back to configurable option, minio completely breaks when tagging is used (overwrites actual file with meta-data file) and testing infrastructure for MT use minio for PR builds

Reply:
yup, I'm online with Greg about it, thx.